src/index.js

```diff
@@ -1,17 +1,24 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+
+import rootReducer from './reducers';
+import App from './components/App';
+
+const defaultBooks = {
+  books: [
+    { id: Math.floor(Math.random() * 10).toString(), title: 'El Viejo y el Mar', category: 'Biography' },
+    { id: Math.floor(Math.random() * 10).toString(), title: 'Harry Potter', category: 'Kids' },
+    { id: Math.floor(Math.random() * 10).toString(), title: 'Something Random', category: 'Learning' },
```

Comment:
Seems like you could get a lot of duplicate ids, maybe a larger range or an [npm package](https://www.npmjs.com/package/uniqid) could help.